### DISCUSSION TITLE
VMM moved?

### DISCUSSION THREAD

**Discussion by Sir-Bacon** [007revad/Synology_app_mover#14] [Q&A] [answered]
Q: I have VMM installed with a single VM running. If I use this script, will it move the complete VM as well?

**Comment by 007revad** [comment, marked as answer]
Don't move VMM just yet.

The new version of this script I'm working on has lots changes to fully, or correctly, move apps that have extra settings or data in folders.

I'm just working on Active Backup for Business and then I'll release the new version. See [Confirmed supported packaged list](https://github.com/007revad/Synology_app_mover/blob/test/README.md)

<br>

**Change log:**

v2.0.6
- Confirmed working for 67 packages.
- **Updated to work correctly with the following packages that have extra folders or symlinks:**
  - Chat, Container Manager, MailPlus Server, Mail Server, Node.js, Presto File Server, Surveillance Station, 
    Synology Application Service, Synology Drive, WebDAV Server, **Virtual Machine Manager**. 
- Moving Container Manager now includes your images and containers.
  - It also asks if you want to backup the @docker folder (just in case).
- Added a progress bar so you know the script is still working on long running processes.
- Now shows the current volume for each package. Issue #12 Thanks woozu-shin
- Now stops (and later starts) dependant packages when moving a package with dependancies.
- Now asks if you want to start the package (in case you want to move it's shared folder first).
- Now instructs how to move packages' shared folder(s) for the following packages:
  - Audio Station, Chat, Cloud Sync, MinimServer, Plex Media Server, Surveillance Station and Video Station.
- **Now instructs how to change the packages' database location for the following packages:**
  - Cloud Sync, Download Station, Note Station, Synology Drive and **Virtual Machine Manager**.
- Now if package is ContainerManager or Docker:
  - Offers to backup @docker.
  - Moves @docker if there's enough space on the target volume.
  - Instructs how to move your docker shared folder.
- Now gets the permissions of the original folder to set on the new target folder.
- Now checks if there's a new version of the script and asks if you want the script to update itself.
- Fixed bug which allowed typing an invalid package selection.
- Fixed bug which allowed typing an invalid volume selection.

**Comment by 007revad** [comment]
I've released the new version: https://github.com/007revad/Synology_app_mover/releases/tag/v2.0.6
